content/en/docs/components/training/admin-guides/installation.md

```diff
@@ -0,0 +1,58 @@
++++
+title = "Installation"
+description = "How to install the Kubeflow Training control plane"
+weight = 10
++++
+
+This guide describes how to install the Kubeflow Training control plane on a Kubernetes cluster.
+
+You can skip these steps if [the Kubeflow platform](https://www.kubeflow.org/docs/started/installing-kubeflow/)
+is already deployed using manifests or package distributions, as it includes Kubeflow Training by default.
+
+## Prerequisites
+
+These are the minimal requirements to install the Kubeflow Training control plane:
+
+- Kubernetes >= 1.28
+- `kubectl` >= 1.28
+
+{{% alert title="Tip" color="primary" %}}
+If you don't have Kubernetes cluster, you can quickly create one locally using [Kind](https://kind.sigs.k8s.io/docs/user/quick-start#installing-with-a-package-manager):
+
+```bash
+brew install kind
+kind create cluster
+```
+
+{{% /alert %}}
+
+## Installing the Kubeflow Training Manager
+
+Run the following command to deploy the Kubeflow Training manager:
+
+TODO (andreyvelich): Change the link once V1 is removed.
+
+```bash
+kubectl apply --server-side -k "github.com/kubeflow/training-operator.git/manifests/v2/overlays/manager?ref=master"
```

Comment:
Don't most copy paste a full URL that starts with https?

Reply:
Actually, this is how you can use remote URL with kubectl and kustomize (e.g. they accept the SSH url: github.com:kubeflow/training-operator.git )